lerna.json

```diff
@@ -1,7 +1,8 @@
 {
   "packages": [
     "packages/*",
-    "packages/samples/**"
+    "packages/samples/**",
+    "packages/atomic-angular/projects/**"
```

Comment:
Why the double `*` at the end? Are there multiple `package.json`?

Reply:
Indeed, `/**` not needed